crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -13,7 +13,7 @@ use std::marker::PhantomData;
 use super::Resource;
 
 /// A [`World`] mutation.
-pub trait Command: Send + Sync + 'static {
+pub trait Command: Send + 'static {
```

Comment:
I think we'll also want to do the same transformation for `Send` at some point.

Note that doing that requires we keep proper main thread management - it requires a lot of care.

Reply:
Is the Send + Sync necessary when the target is wasm?